narwhals/_dask/expr.py

```diff
@@ -545,6 +545,22 @@ def func(_input: Any) -> Any:
             returns_scalar=False,
         )
 
+    def is_duplicated(self: Self) -> Self:
+        msg = "`Expr.is_duplicated` is not support since Dask currently has no native duplicated check"
+        raise NotImplementedError(msg)
+
+    def is_unique(self: Self) -> Self:
+        msg = "`Expr.is_duplicated` is not support since Dask currently has no native duplicated check"
```

Comment:
I am late but

```diff
-        msg = "`Expr.is_duplicated` is not support since Dask currently has no native duplicated check"
+        msg = "`Expr.is_unique` is not support since Dask currently has no native duplicated check"

Reply:
aah crap, thanks - fancy bundling this in with another one?

Reply:
Sure!

Reply:
Doh! Sorry both 🤦‍♂️

Reply:
No worries! I tend to say that until a release we are fine to mess up a bit 😂